integration-scripts/install_cppstats.sh

```diff
@@ -14,7 +14,7 @@ echo '#!/bin/bash' > $CPPSTATS/cppstats
 echo "cd $CPPSTATS" >> $CPPSTATS/cppstats
 echo "PYTHONPATH=\"\$PYTHONPATH:$CPPSTATS/lib\" ./cppstats.py \"\$@\"" >> $CPPSTATS/cppstats
 chmod +x $CPPSTATS/cppstats
-wget --quiet http://sdml.info/lmcrs/srcML-Ubuntu12.04-64.tar.gz -O /tmp/srcML.tar.gz
+wget --quiet http://131.123.42.38/lmcrs/beta/srcML-Ubuntu12.04-64.tar.gz -O /tmp/srcML.tar.gz
```

Comment:
The download location of choice is ``http://131.123.42.38/lmcrs/srcML-Ubuntu12.04-64.tar.gz``, as cppstats is _not_ compatible with the current beta release of srcML.